Resources/doc/index.md

```diff
@@ -32,7 +32,7 @@ framework:
     esi: { enabled: true } # use to reload csrf token
 
 sulu_form:
-    mail_helper:
+    mail:
         from: %parameter_recommended_for_from%
```

Comment:
we could change this to `sulu_admin.email` so it is working on the fly without adding a new parameters to the .yml file